tools/cmake/README.md

```diff
@@ -75,3 +75,24 @@ If you're running CMake directly, you may need to pass it in yourself as follows
 ```
 cmake -S <source-dir> -B <build-dir> -DCMAKE_BUILD_TYPE=debug
 ```
+
+## How to build a greentea test
+
+Install prerequisites suggested in the previous section and follow the below steps to build:
+* Generate the `.mbedbuild/` configuration directory for the Mbed target you want to run the test on using [mbed-os-example-blinky](https://github.com/ARMmbed/mbed-os-example-blinky)
+``'
```

Comment:
This needs fixing I put the wrong stuff on purpose for formatting reasons in my comment.